pipelines/batch/src/main/java/com/google/fhir/analytics/JdbcFetchHapi.java

```diff
@@ -157,7 +157,8 @@ public FetchRowsJdbcIo(
       StringBuilder builder =
           new StringBuilder(
               "SELECT res.res_id, hfi.forced_id, res.res_type, res.res_updated, res.res_ver,"
-                  + " res.res_version, ver.res_encoding, ver.res_text, ver.res_text_vc  FROM hfj_resource res JOIN"
+                  + " res.res_version, ver.res_encoding, ver.res_text, ver.res_text_vc "
+                  + " FROM hfj_resource res JOIN"
```

Comment:
nit: looks like the formatting is not done for this line.

Reply:
Actually the formatting is fine here (just ran `mvn package` again); I intentionally moved the `FROM` (and now the `WHERE`) to the start of the line to make the query more readable.